WebApiCoreSeed.UnitTests/Services/UserServiceTests.cs

```diff
@@ -0,0 +1,52 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Threading.Tasks;
+using WebApiCoreSeed.Data.Models;
+using WebApiCoreSeed.Domain.Services;
+using WebApiCoreSeed.Domain.Services.Interfaces;
+
+namespace WebApiCoreSeed.UnitTests.Services
+{
+    [TestClass]

```

Comment:
Microsoft is still maintaining MSTests, but it seems that cool projects are using xUnit. Did you consider to use xUnit in place of MSTest?

See a related discussion in Slack: https://slarck.com/usy2uI/be-devs/2017/04/20170404184516#page-wrapper

![2017-05-15_133236](https://cloud.githubusercontent.com/assets/1157864/26068178/06f093d0-3973-11e7-8798-5b073b5b1f59.png)